<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>integration</class>
 <widget class="QMainWindow" name="integration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>1321</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab {
	background-color: rgba(126, 75, 255,50%);
  color: white;
  padding: 10px 40px;
border-radius:18px;
font: 9pt &quot;MS Shell Dlg 2&quot;;
}

QTabBar::tab:selected {
    color: white;
   background-color:#FF7244;

}
QTabBar::tab:hover {
  background: #FF7244;
 }


QTabWidget::pane {
  top:0px; 
	background-color: rgba(126, 75, 255,50%);
border-radius:10px;
font: 12pt &quot;MS Shell Dlg 2&quot;;
} 
QLineEdit{
border-radius:15px;
padding:4px;
padding-left:10px;
padding-right:15px;
color:#7375f3;

font:  8pt &quot;MS Shell Dlg 2&quot;;
}
QLabel{
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion employes</string>
     </attribute>
     <widget class="QPushButton" name="pb_excel">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pb_excel{
background-color:transparent;
border-radius:15px;
}
#pb_excel:hover{
background-color:#FF7244;
border-radius:10px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/excel.png</normaloff>../../Downloads/excel.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="stat">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>370</y>
        <width>511</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>0</y>
        <width>311</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../projetqt/Fichier 1.png</normaloff>../projetqt/Fichier 1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="qr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgba(126, 75, 255,80%);  color: white;
  padding: 10px 12px;
border-radius:17px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton::selected {
    color: white;
   background-color:#FF7244;

}
QPushButton::hover {
  background: #FF7244;
 }</string>
      </property>
      <property name="text">
       <string>qr code</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>340</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Trier par:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
padding:4px;
padding-left:10px;
padding-right:15px;
color:#7375f3;
font-size:15px;
font-weight:bold;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string extracomment="Recherche">Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton{
background-color:transparent;
border-radius:15px;
}
#pushButton:hover{
background-color:#FF7244;
border-radius:10px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/774684_pdf_document_extension_file_format_icon (1).png</normaloff>../../Downloads/774684_pdf_document_extension_file_format_icon (1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_stat">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>420</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgba(126, 75, 255,80%);  color: white;
  padding: 10px 12px;
border-radius:17px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton::selected {
    color: white;
   background-color:#FF7244;

}
QPushButton::hover {
  background: #FF7244;
 }</string>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nom">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>330</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgba(126, 75, 255,80%);  color: white;
  padding: 10px 12px;
border-radius:17px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton::selected {
    color: white;
   background-color:#FF7244;

}
QPushButton::hover {
  background: #FF7244;
 }</string>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>431</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab {
	background-color: rgba(126, 75, 255,50%);
  color: white;
  padding: 10px 40px;
border-radius:18px;
font: 9pt &quot;MS Shell Dlg 2&quot;;
}

QTabBar::tab:selected {
    color: white;
   background-color:#FF7244;

}
QTabBar::tab:hover {
  background: #FF7244;
 }


QTabWidget::pane {
  top:0px; 
	background-color: rgba(126, 75, 255,50%);
border-radius:10px;
font: 12pt &quot;MS Shell Dlg 2&quot;;
} 
QLineEdit{
border-radius:15px;
padding:4px;
padding-left:10px;
padding-right:15px;
color:#7375f3;

font:  8pt &quot;MS Shell Dlg 2&quot;;
}
QLabel{
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>47</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nom">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>120</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Prénom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>120</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>160</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Post</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QComboBox" name="le_post">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>160</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white;
color:#585856;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>370</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(126, 75, 255,80%);  color: white;
  padding: 10px 12px;
border-radius:17px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton::selected {
    color: white;
   background-color:#FF7244;

}
QPushButton::hover {
  background: #FF7244;
 }</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>210</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_email">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>210</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>250</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_password">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>250</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>290</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Salaire</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_salaire">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>290</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>330</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Num Tel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_tel">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>330</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>210</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(126, 75, 255,80%);  color: white;
  padding: 10px 12px;
border-radius:17px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton::selected {
    color: white;
   background-color:#FF7244;

}
QPushButton::hover {
  background: #FF7244;
 }</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>140</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_idS">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>170</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QPushButton" name="pb_modifier">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>360</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>9</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(126, 75, 255,80%);  color: white;
  padding: 10px 12px;
border-radius:17px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton::selected {
    color: white;
   background-color:#FF7244;

}
QPushButton::hover {
  background: #FF7244;
 }
</string>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>120</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Prénom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nom_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>160</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Post</string>
        </property>
       </widget>
       <widget class="QComboBox" name="le_post_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>160</y>
          <width>162</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white;
color:#585856;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_email_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>200</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>280</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Salaire</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>240</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>320</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Num Tel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_salaire_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>280</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_31">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>200</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_password_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>240</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_tel_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>320</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>historique</string>
       </attribute>
       <widget class="QTextBrowser" name="textHisto">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>431</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pb_rechercher">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>40</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#pb_rechercher{
background-color:white;
border-radius:15px;
}

#pb_rechercher:hover{
background-color:#D6DBFD;

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../search.png</normaloff>../search.png</iconset>
      </property>
     </widget>
     <widget class="QTableView" name="tab_employe">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>80</y>
        <width>721</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#tab_employe{
	border:3px solid rgb(101, 104, 243);
	border-radius: 50px;
	background-color: rgba(126, 75, 255,50%);
	
}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="qrcode">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>520</y>
        <width>171</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="id">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>330</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgba(126, 75, 255,80%);  color: white;
  padding: 10px 12px;
border-radius:17px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton::selected {
    color: white;
   background-color:#FF7244;

}
QPushButton::hover {
  background: #FF7244;
 }</string>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <zorder>tabWidget_2</zorder>
     <zorder>qrcode</zorder>
     <zorder>pb_rechercher</zorder>
     <zorder>id</zorder>
     <zorder>pushButton</zorder>
     <zorder>recherche</zorder>
     <zorder>stat</zorder>
     <zorder>qr</zorder>
     <zorder>pb_stat</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>pb_excel</zorder>
     <zorder>label_6</zorder>
     <zorder>nom</zorder>
     <zorder>tab_employe</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion 2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Gestion 3</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Gestion 4</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1312</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
